e2e/interchaintestv8/chainconfig/genesis.go

```diff
@@ -15,7 +15,7 @@ import (
 	"github.com/srdtrk/solidity-ibc-eureka/e2e/v8/testvalues"
 )
 
-func defaultModifyGenesis() func(ibc.ChainConfig, []byte) ([]byte, error) {
+func DefaultModifyGenesis() func(ibc.ChainConfig, []byte) ([]byte, error) {
```

Comment:
Why did we make this public? (Not that it matters)

Reply:
Yeah, that was probably during some early experimentation, will revert this